cypress/e2e/pages/UserDashboard.js

```diff
@@ -4,65 +4,104 @@ import Loaders from "../components/Loaders";
 class UserDashboard extends BasePage {
 
   // Locators    
-  get workExperience() { return cy.get("h2[id='heading4'] button[type='button']") }
-  get noRecordAvailable() { return cy.get("#collapse4 div.table-responsive div") }
-  get basicInfo() { return cy.get("h2[id='heading1'] button[type='button']") }
-  get editBasicInfo() { return cy.get("#collapse1 a i") }
-  get closeButton() { return cy.get("button[class='btn btn-secondary btn-sm ms-2']") }
-  get editFirstName() { return cy.get("input[value='Auto']") }
-  get editLastName() { return cy.get("input[value='User']") }
-  get editMiddleName() { return cy.get("input[value='Mation']") }
-  get updateButton() { return cy.get("button[class='btn btn-primary btn-sm']") }
+  get workExperienceAccord() { return cy.get("h2[id='heading4'] button") }
+  get noRecordAvailableInfo() { return cy.get("#collapse4 div.table-responsive div") }
+  get basicInfoAccord() { return cy.get("h2[id='heading1'] button") }
+  get editBasicInfoBtn() { return cy.get("#collapse1 a i") }
+  get closeBtn() { return cy.get("button[class='btn btn-secondary btn-sm ms-2']") }
+  get editFirstNameTxt() { return cy.get("input[name='firstName']:not(:disabled)") }
+  get editLastNameTxt() { return cy.get("input[name='lastName']:not(:disabled)") }
+  get editMiddleNameTxt() { return cy.get("input[name='middleName']:not(:disabled)") }
+  get updateBtn() { return cy.get("button[class='btn btn-primary btn-sm']") }
+  get workAccord() { return cy.get("h2[id='heading2'] button") }
+  get editWorkInfoBtn() { return cy.get('#collapse2 a i') }
+  get editDateOfJoiningPicker() { return cy.get("div[class='input-group'] [class='border']") }
+  get personalDetailsAccord() { return cy.get("h2[id='heading3'] button") }
+  get editDateOfBirthPicker() { return cy.get("#collapse3 input[type='date']") }
+  get editPassportNumberTxt() { return cy.get("input[name='passportNumber']:not(:disabled)") }
+  get editPanNumberTxt() { return cy.get("input[name='panCardNumber']:not(:disabled)") }
+  get editAdhaarNumberTxt() { return cy.get("input[name='aadharNumber']:not(:disabled)") }
+  get editPresentAddressTxt() { return cy.get("textarea[name='presentAddress']:not(:disabled)") }
+  get editPermanentAddressTxt() { return cy.get("textarea[name='permanentAddress']:not(:disabled)") }
+  get editAlternateNumberTxt() { return cy.get("input[name='alternateNumber']:not(:disabled)") }
+  get editBloodGroupDrp() { return cy.get("div [id='bloodGroup']:not(:disabled)") }
+  get editMaritalStatusDrp() { return cy.get("select[id='maritalStatus']:not(:disabled)") }
+  get editGenderBtn() { return cy.get("#female:not(:disabled)") }
 
   // Methods
-  clickOnWorkExperience() {
-    this.workExperience.click();
+  clickOnWorkExperienceAccord() {
```

Comment:
No need to define postfix like Accord in method name

Reply:
Resolved